### PR TITLE
Update committer page.

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
This PR aims to update `Project Committers' page before 2.2 release event.

### CODE REVIEW

File: committers.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ navigation:
 |----|------------|
 |Michael Armbrust|Databricks|
 |Joseph Bradley|Databricks|
-|Felix Cheung|Automattic|
```
Comment (dongjoon-hyun) [inline committers.md:15]: Hi, @felixcheung .
I guess this is right. Could you confirm this?

File: committers.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ navigation:
 |Andy Konwinski|Databricks|
 |Ryan LeCompte|Quantifind|
 |Haoyuan Li|Alluxio, UC Berkeley|
-|Xiao Li|IBM|
```
Comment (dongjoon-hyun) [inline committers.md:35]: Hi, @gatorsmile .
Could you confirm this? :)

File: committers.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ navigation:
 |Prashant Sharma|IBM|
 |Ram Sriharsha|Databricks|
 |DB Tsai|Netflix|
-|Takuya Ueshin||
```
Comment (dongjoon-hyun) [inline committers.md:54]: Hi, @ueshin .
Could you confirm this?

**Comment by srowen** [comment]
I know Sandy is at a company called Remix now too

**Comment by dongjoon-hyun** [comment]
Thank you! I'll fix that too.

File: committers.md (hunk 3 above)
Comment (ueshin) [reply committers.md:54]: Yes, that's right.

Comment (dongjoon-hyun) [reply committers.md:54]: Thank you! :)

**Comment by felixcheung** [comment]
Yes thanks!

**Comment by dongjoon-hyun** [comment]
Thank you, @felixcheung !

File: committers.md (hunk 2 above)
Comment (gatorsmile) [reply committers.md:35]: Yes

Comment (dongjoon-hyun) [reply committers.md:35]: Thank you, @gatorsmile . :-)

**Comment by dongjoon-hyun** [comment]
Hi, @srowen .
All updated information are confirmed.
Could you review again?

**Review by srowen — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
Thank you, @srowen .

**Comment by dongjoon-hyun** [comment]
Thank you for merging! :)

**Pull request merged**